### DISCUSSION TITLE
Start app with hidden main window

### DISCUSSION THREAD

**Discussion by fcoury** [tauri-apps/tauri#5364] [General]
What's the easiest way to prevent the main window from opening on a SystemTray-based app?

**Comment by amrbashir** [comment]
remove it from `tauri.conf.json > tauri > windows` or add `"vsibile": false` to it, if you remove it, you need to prevent app exit like this https://github.com/tauri-apps/tauri/blob/4cbdf0fb1c0de5004eab51c36d5843a9816f18af/examples/api/src-tauri/src/lib.rs#L146-L150

**Reply by fcoury** [reply]
Thank you!
